glide.yaml

```diff
@@ -65,4 +65,19 @@ import:
   version: 0fe204460da2c8fa1babcaac196e694de8f1aaa1
 
 - package: github.com/miekg/dns
-  version: db96a2b759cdef4f11a34506a42eb8d1290c598e
\ No newline at end of file
+  version: db96a2b759cdef4f11a34506a42eb8d1290c598e
+
+- package: github.com/shirou/gopsutil
+  version: 2.1.0
+  subpackages:
+  - load
+
+- package: github.com/davecgh/go-spew
```

Comment:
Why do we need to lock down a specific version of `github.com/davecgh/go-spew`? We only use it indirectly via `github.com/stretchr/testify`, which already vendors it.